src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,9 @@
+.catalog {
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  width: 1000px;
```

Comment:
**Critical mistake!**

**Width** of catalog can not be fixed, it should be dictated by cards in catalog row.
I suggest you to use calc() function and next expression:

`Width = (number of cards in a row) * (card width) + (number of cards in a row - 1) * (gap between cards)`